packages/design-system/collapsible-tab-view/gesture-container.ios.tsx

```diff
@@ -615,7 +617,9 @@ export const GestureContainer = React.forwardRef<
         <GestureDetector gesture={gestureHandlerHeader}>
           <Animated.View style={styles.container}>
             {renderScrollHeader && (
-              <View onLayout={headerOnLayout}>{renderScrollHeader()}</View>
+              <View onLayout={headerOnLayout}>
+                {renderScrollHeader(translateYValue)}
```

Comment:
if we use `renderScrollHeader({ translateYValue })` replace with it, would scalability be better?

but now it's not a blocker, just not sure if other parameters ​​will be passed later.